### ISSUE TITLE
documentation of the final product

### ISSUE THREAD

**Issue by JonatanS** [JonatanS/dunit#5]
Once the viewer is fixed, I will document this in form of a screen capture. @ksobon maybe you can help with this, since you have all of the dynamo files. We could just meet in one of our offices and record it... A few stills would be good, too.

**Comment by ksobon** [comment]
Yeah! That's a good idea. I will try to find some time next week to post all of the Dynamo stuff...I came into the office today and landed on a competition team with a short deadline so I am strapped for a little bit (Next week).
